src/Tickets/Commerce/Memoize_Attendees.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Memoize Attendees class is a helper class for the Tickets List Table to reduce the number of queries.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Admin\Commerce
+ */
+
+namespace TEC\Tickets\Commerce;
+
+/**
+ * Class Memoize_Attendees
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Admin\Commerce
+ */
+class Memoize_Attendees {
```

Comment:
I am wondering if we need an extra class for this, why no use `tribe_cache()` and store as two separate arrays there, it might make it easier for tests because we can tell it to ignore cache.

Reply:
The idea came out of a discussion we were having about it. We can use `tribe_cache()` though. No problem.